### DISCUSSION TITLE
Complete example with the DSL

### DISCUSSION THREAD

**Discussion by eduardoconto** [Safe-DS/.github#8] [DSL Q&A]
Hello. Interesting concept & language. Do you have working examples of pipelines implemented in the DSL (for example, with the "Titanic" dataset referred to in your Safe-DS 2023 paper)? Or others showcasing the different features of the language? Thanks.

**Comment by lars-reimann** [comment]
Thanks for your interest in the project. This part of the documentation depends on a few other developments, which we need to tackle first:

1. Continuation of the [stub generator](https://github.com/Safe-DS/Stub-Generator) to automatically create _stubs_ (interface descriptions) for our [Python library](https://github.com/Safe-DS/Stdlib), so it can be used inside the DSL.
2. Creation of stubs for the [Python library](https://github.com/Safe-DS/Stdlib). At this stage, we still might need to make slight manual changes to the generated stubs, but these will be reduced in the future with improvements to the stub generator.

Once that is done, we've reached a status where the DSL can be used for actual ML workflows. Then, we'll also update the documentation accordingly with proper examples. The documentation for _concepts_ of the pipeline language is already available [here](https://dsl.safeds.com/en/stable/language/pipeline-language/).

**Reply by lars-reimann** [reply]
An update on this: The stubs are now shipped with `v0.9.0`. There is also documentation for the [builtin library](https://dsl.safeds.com/en/stable/stdlib/safeds/data/image/containers/Image/). We are currently squashing some bugs and will then add example pipelines to the documentation.
